dev/bench/bench.sh

```diff
@@ -316,9 +319,12 @@ create_opam() {
 
     local RUNNER="$1"
     local OPAM_DIR="$working_dir/opam.$RUNNER"
-    local OPAM_COMP="$2"
+    local OCAML_VER="$2"
     local COQ_HASH="$3"
     local OPAM_COQ_DIR="$4"
+    local USE_FLAMBDA="$5"
+
+    local OPAM_COMP=ocaml-base-compiler.$new_ocaml_version
```

Comment:
:skull: 
so all these benches were testing NEW (which I called OLD in the commit messages) against itself

Reply:
Except the first one, then. How didn't you screw that one but all the other?

Reply:
this was modified in the commit to test flambda